src/Cli.Tests/EndToEndTests.cs

```diff
@@ -517,118 +503,7 @@ public void TestEngineStartUpWithVerboseAndLogLevelOptions(string logLevelOption
         DisplayName = "Correct command with correct options should have exit code 0.")]
     public void VerifyExitCodeForCli(string[] cliArguments, int expectedErrorCode)
     {
-        Assert.AreEqual(Cli.Program.Main(cliArguments), expectedErrorCode);
-    }
-
-    /// <summary>
-    /// Test to verify that help writer window generates output on the console.
-    /// </summary>
-    [DataTestMethod]
-    [DataRow("", "", new string[] { "ERROR" }, DisplayName = "No flags provided.")]
-    [DataRow("initialize", "", new string[] { "ERROR", "Verb 'initialize' is not recognized." }, DisplayName = "Wrong Command provided.")]
-    [DataRow("", "--version", new string[] { "Microsoft.DataApiBuilder 1.0.0" }, DisplayName = "Checking version.")]
-    [DataRow("", "--help", new string[] { "init", "add", "update", "start" }, DisplayName = "Checking output for --help.")]
-    public void TestHelpWriterOutput(string command, string flags, string[] expectedOutputArray)
-    {
-        using Process process = ExecuteDabCommand(
-            command,
-            flags
-        );
-
-        string? output = process.StandardOutput.ReadToEnd();
-        Assert.IsNotNull(output);
-        Assert.IsTrue(output.Contains($"{Program.PRODUCT_NAME} {GetProductVersion()}"));
-
-        foreach (string expectedOutput in expectedOutputArray)
-        {
-            Assert.IsTrue(output.Contains(expectedOutput));
-        }
-
-        process.Kill();
-    }
-
-    /// <summary>
-    /// Test to verify that the version info is logged for both correct/incorrect command,
-    /// and that the config name is displayed in the logs.
-    /// </summary>
-    [DataRow("", "--version", false, DisplayName = "Checking dab version with --version.")]
-    [DataRow("", "--help", false, DisplayName = "Checking version through --help option.")]
-    [DataRow("edit", "--new-option", false, DisplayName = "Version printed with invalid command edit.")]
-    [DataRow("init", "--database-type mssql", true, DisplayName = "Version printed with valid command init.")]
-    [DataRow("add", "MyEntity -s my_entity --permissions \"anonymous:*\"", true, DisplayName = "Version printed with valid command add.")]
-    [DataRow("update", "MyEntity -s my_entity", true, DisplayName = "Version printed with valid command update.")]
-    [DataRow("start", "", true, DisplayName = "Version printed with valid command start.")]
-    [DataTestMethod]
-    public void TestVersionInfoAndConfigIsCorrectlyDisplayedWithDifferentCommand(
```

Comment:
1. TestHelpWriterOutput
2. TestEngineStartUpWithVerboseAndLogLevelOptions
3. TestExitOfRuntimeEngineWithInvalidConfig

Are all removed, can you clarify why? it isn't immediately clear whetehr these are moved or deleted completely.